app/src/main/res/layout/preview__activity.xml

```diff
@@ -1,16 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-              android:layout_width="match_parent"
-              android:layout_height="match_parent"
-              android:orientation="vertical">
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
 
     <include
         android:id="@+id/toolbar"
-        layout="@layout/toolbar"/>
+        layout="@layout/toolbar" />
 
-    <WebView
-        android:id="@+id/preview__activity__webview"
+    <ScrollView
```

Comment:
Does scrolling work on older devices (minApi) too? Horizontal and vertical? <https://stackoverflow.com/questions/9718245/webview-in-scrollview>

Reply:
I just did other tests varying the implementation and also the devices, so:

1. It works with the current minApi (16)
2. As you can see, I did two commits, the ScrollView was needed to make the first implementation work. I have tested the pull request's current implementation and the ScrollView is not needed anymore

Do I create another commit without the ScrollView?